graphql/resolve/add_mutation_test.yaml

```diff
@@ -1278,6 +1278,16 @@
         uid
       }
     }
+  dgquerysec: |-
+    query {
+      LibraryMember_1 as LibraryMember_1(func: uid(0x11)) @filter(type(LibraryMember)) {
+        uid
+      }
+    }
+  qnametouid: |-
+    {
+      "LibraryMember_1": "0x11"
+    }
```

Comment:
nit: keep the qnametouid field above dgquerysec as done in other test cases. It will then be easier to follow the test case.